scans/worker.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+
+from scans import decimalencoder
+import boto3
+dynamodb = boto3.resource('dynamodb')
+from scans import Response
+
+
+def worker(event, context):
+    table = dynamodb.Table(os.environ['DYNAMODB_TABLE'])
+
+    # Example event to play with
+    # {
+    #     "Records": [
```

Comment:
I've seen a few examples of this data structure in docs, and it varied, probably multiple version of the schema.  I think the best way to be sure is setup the handler and capture an example event to make sure we're using the latest format.

Reply:
The schema that comes out is a standard dynamoDb stream event.  The OLD_NEW_IMAGE format is based on the schema that you form in dynamodb yourself.  Event format never changes.